### PR TITLE
[SDK] Add collider support to prefabs

### LINKED ISSUE #110: AssetManager.loadGltf results in unexpected reply message
It seems the reply message isn't getting routed to the pending promise.

### PR DESCRIPTION
**Pull request by eanders-MS**
AssetManager.loadGltf now accepts a `colliderType` parameter, allowing for colliders on preloaded assets. Correspondingly, Actor.CreateFromPrefab has a new parameter called `enableColliders` to enable them when instancing a prefab.

Also: Improved Actor.Create* jsdocs.

Fixes #110 and #111

### CODE REVIEW

**Comment by eanders-MS** [comment]
@tombuMS This just brings Actor.CreateFromPrefab up to par with the Actor.CreateFromGltf and Actor.CreatePrimitive functions, where the API already supports colliders. We can deprecate this API once colliders are in if it makes sense to do so.

What does the collider API look like on the Node side? It would be helpful to review it and discuss how it will change the Actor.Create* functions.

File: packages/sdk/src/types/internal/context.ts [hunk 1]
```diff
@@ -223,10 +223,12 @@ export class InternalContext {
     public CreateFromPrefab(options: {
         prefabId: string,
         actor?: Partial<ActorLike>,
+        enableColliders?: boolean,
```
[the remark this answers is not in the document]
Comment (eanders-MS) [reply packages/sdk/src/types/internal/context.ts]: I struggled naming this one. Maybe the plurality of the name is leaking an implementation detail, but prefabs are glTFs, and as far as I know the glTF loader will add a collider to any node that has a mesh, so there can be more than one collider in the prefab.

File: packages/sdk/src/adapters/multipeer/client.ts [hunk 2]
```diff
@@ -159,6 +159,6 @@ export class Client extends EventEmitter {
     public static ShouldIgnorePayloadWhileJoining(type: string): boolean {
         // Ignore "create" payloads while joining, since all created things will be synchronized
         // from the ClientSync protocol.
-        return type.startsWith('create-');
+        return type.startsWith('create-') || type.startsWith('load-');
```
[the remark this answers is not in the document]
Comment (eanders-MS) [reply packages/sdk/src/adapters/multipeer/client.ts]: Good eye! You're correct. I have a fix coming for this in a different PR. It's part of my fix for #104, where this method has changed substantially. This is a stop-gap. I think it's ok for now.

[the remark this answers is not in the document]
Comment (stevenvergenz) [reply packages/sdk/src/adapters/multipeer/client.ts]: In that case, I think we should attempt to merge these PRs together. Unless loadAssets is broken for late joiners already?

[the remark this answers is not in the document]
Comment (eanders-MS) [reply packages/sdk/src/adapters/multipeer/client.ts]: Fixed it.

**Pull request merged**